docs/40_topics/01_fair.mdx

```diff
@@ -7,37 +7,90 @@ author: "Nicole A. Parks"
+is the primary example of the FAIR Data Principles in action.
+To broaden the scope of such standards, [NFDI4Chem](https://www.nfdi4chem.de/) is working towards creating an infrastructure
+for FAIR data in Germany while training researchers in using the available infrastructure to ensure their data is as FAIR as can be.
+
+In the following, we answer the questions:
+What makes data FAIR?
+What do researchers and those who provide data preservation services need to consider?
 
 ## Findable
 
-Researchers – and the computers working on their behalf – must be able to find datasets to be able to reuse them. Therefore, the first guideline of the FAIR Data Principles outlines methods to ensure a dataset's discovery.
+Researchers — and the computers working on their behalf –
+must be able to find datasets to be able to reuse them.
+Therefore, the first guideline of the FAIR Data Principles outlines methods to ensure a dataset’s discovery.
 
 ### F1. (meta)data are assigned a globally unique and persistent identifier
 
-A globally unique and persistent identifier (PID) helps both machines and humans find the data in the first place. The registry services that make these identifiers available work to maintain the link to the resource they are associated with, in this case, a dataset. A common example is the Digital Object Identifier, or [DOI](https://doi.org/10.1000/182). This type of identifier ensures the object it refers to, such as a dataset or a journal publication, are not only findable but also citable. As with many journals, scientific data repositories oftentimes assign a DOI automatically. The Registry of Research Data Repositories, [re3data](https://www.re3data.org/), indicates whether a given repository assigns an identifier, along with the PID type. For example, both the [The Cambridge Structural Database (CSD)](https://www.ccdc.cam.ac.uk/solutions/csd-system/components/csd/) and the [Chemotion Repository](https://www.chemotion-repository.net/) assign DOIs to each dataset deposited. Researchers must be aware of this option when searching for a suitable repository, while repositories should offer this service.
+A globally unique and persistent identifier (PID) helps both machines and humans find the data in the first place.
+The registry services that make these identifiers available work to maintain the link to
+the resource they are associated with, in this case, a dataset.
+A common example is the Digital Object Identifier, or [DOI](https://doi.org/10.1000/182).
+This type of identifier ensures the object it refers to,
+such as a dataset or a journal publication,
+are not only findable but also citable.
+
+<!-- Ich würde hier nochmals herausstellen, was ein PID so besonders
+und für die Forschung essentiell macht: Die Garantie der
+PID-Servicestellen, dass die hinter einer PID/DOI hinterlegten Daten
+stets verfügbar und referenzierbar sind und dass sich die Daten
+nicht ändern (ansonsten gibt es eine Version und eine neue PID/DOI).-->
+
```

Comment:
Stimmt das aber immer? Beispielsweise, in dem Fall von [accepted manuscripts bei der RSC](https://www.rsc.org/journals-books-databases/author-and-reviewer-hub/process-and-policies/#accepted-manuscripts) wird ein DOI für die accepted Version erteilt, derselbe aber auch für die finale Version benutzt.